openreview/conference/matching.py

```diff
@@ -466,6 +466,12 @@ def _build_config_invitation(self, scores_specification):
                         'value-regex': '.*',
                         'required': False,
                         'order': 17
+                    },
+                    'allow_all_affinities': {
```

Comment:
I would like something more description, like 'allow_zero_score_assignments` and a description of this parameter.